components/concept/storage/src/main/java/mozilla/components/concept/storage/CreditCardsAddressesStorage.kt

```diff
@@ -362,7 +362,17 @@ data class Address(
     val timeLastUsed: Long? = 0L,
     val timeLastModified: Long = 0L,
     val timesUsed: Long = 0L
-) : Parcelable
+) : Parcelable {
+    /**
+     * Returns the full name for the [Address]. The combination of names is based on desktop code
+     * found here:
+     * https://searchfox.org/mozilla-central/rev/d989c65584ded72c2de85cb40bede7ac2f176387/toolkit/components/formautofill/FormAutofillNameUtils.jsm#400
+     */
+    val fullName: String
```

Comment:
This is upstream from Fenix https://github.com/mozilla-mobile/fenix/commit/483f5a9f125ba8613dd2e29914e39a1f4e293847#diff-b86bfb8c9a86ae7e71af1ffae0a2aa60a26d1be59a2d66e9a334684a91f243bc